frontend/src/metabase/css/core/overlays/utils.ts

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line no-restricted-imports
+import { within } from "@storybook/testing-library";
+
+import { hidden } from "./constants";
+
+export const findListboxWithOption = async (
+  portalRoot: HTMLElement,
+  optionText: string | RegExp,
+) => {
+  const listboxes = await within(portalRoot).findAllByRole("listbox", hidden);
+
+  for (const listbox of listboxes) {
+    try {
+      await within(listbox).findByRole("option", {
+        name: optionText,
+        ...hidden,
+      });
+      return listbox;
+    } catch (error) {
+      // continue
```

Comment:
can we log the error so it doesn't get swallowed?

Reply:
This isn't a true error - it's not an unexpected behavior. This is just the only way I know how to find a listbox containing a particular option